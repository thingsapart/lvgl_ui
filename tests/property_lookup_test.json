[
  {
    "type": "label",
    "id": "@test_label",
    "named": "my_test_label",
    "text": "Direct Prop Test",
    "width": 200,
    "align": "LV_ALIGN_CENTER"
  }
]
