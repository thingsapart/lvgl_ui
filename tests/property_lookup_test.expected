// Code generated by LVGL UI Generator

#include "lvgl.h"

// Forward declaration for any registered C pointers you might have from your application.
// Example: extern const lv_font_t my_custom_font;
void create_ui(lv_obj_t* parent) {
    {
        lv_obj_t* my_test_label = lv_label_create(parent);
        lv_label_set_text(my_test_label, "Direct Prop Test");
        lv_obj_set_style_width(my_test_label, 200);
        lv_obj_set_style_align(my_test_label, LV_ALIGN_CENTER);
        registry_add_pointer(ui_registry, my_test_label, "test_label", "label");
    }
}
